PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+## Pull Request Guidelines for Authors
+
+Summary of [RFC66 - Pull Request & Code Review Standards](https://safetyculture.atlassian.net/wiki/spaces/ENG/pages/2881716914/RFC66+Pull+Request+Code+Review+Standards):
```

Comment:
I think these should be comments and hidden in the final output, since the body of the PR is mainly useful for reviewers.